R/sprintf_linter.R

```diff
@@ -105,9 +104,7 @@ sprintf_linter <- function() {
   }
 
   Linter(linter_level = "file", function(source_expression) {
-    xml <- source_expression$full_xml_parsed_content
```

Comment:
why drop this local variable?

I would keep `xml_calls <- .$xml_find_function_calls(.)`. Having this variable stored is very helpful for debugging+development IME.